src/i18n/locales/cs/translation.json

```diff
@@ -571,6 +571,32 @@
         "description": "",
         "no_info": ""
       },
+      "tips": {
```

Comment:
No need to add these empty ones. [Weblate](https://hosted.weblate.org/git/getalby-lightning-browser-extension/getalby-lightning-browser-extension/) will take care of these.

Reply:
oh, thats cool. I've removed them.